integration_tests/config_store/main_test.go

```diff
@@ -60,4 +61,25 @@ func TestConfigStore(t *testing.T) {
+	if got != want {
+		t.Errorf("Body = %q, want %q", got, want)
+	}
+
+	maxUTF8, err := d.Get(strings.Repeat("ゝ", 255))
```

Comment:
Haven't seen this character in a long time! :)

Reply:
Had to find a good one that was 3 bytes in UTF-8 and the repetition mark seemed like a good fit here. 🍺